src/Files.App/Actions/Content/Share/ShareItemAction.cs

```diff
@@ -0,0 +1,135 @@
+﻿using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.DependencyInjection;
+using Files.App.Commands;
+using Files.App.Contexts;
+using Files.App.Extensions;
+using Files.App.Filesystem;
+using Files.App.Filesystem.StorageItems;
+using Files.App.Helpers;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Windows.ApplicationModel.DataTransfer;
+using Windows.Foundation;
+using Windows.Storage;
+
+namespace Files.App.Actions.Content.Share
+{
+	internal class ShareItemAction : ObservableObject, IAction
+	{
+		private readonly IContentPageContext context = Ioc.Default.GetRequiredService<IContentPageContext>();
+
+		public string Label => "Share".GetLocalizedResource();
+
+		public string Description => "TODO: Need to be described.";
+
```

Comment:
Can you add a description so we don't have to come back to this in the future?

Reply:
Descriptions will be added later, it should be left like this for now.